src/channel-list-item.tsx

```diff
@@ -57,8 +64,16 @@ export class ChannelViewModel extends Model {
     }, undefined);
   }
 
-  private getDisplayName(name: string) {
-    return name.length < 25 ? name : `${name.substr(0, 25)}...`;
+  private getDisplayName(c: ChannelBase): Observable<string> {
+    // NB: This Feels Bad. We should be encouraging people to *create*
```

Comment:
We should make an intermediate property that is the relevant `user` and then both `profileImage` and `displayName` can be based on that.